index.html

```diff
@@ -2,12 +2,190 @@
+                target="_blank"
+                rel="noopener noreferrer"
+                ><img
+                  src="beat-that.png"
+                  alt="Beat That"
+                  width="200"
+                  height="200"
+                  class="graphics"
+              /></a>
+              <p class="anno-width">Simple beat-that game for 2 players.</p>
+            </div>
+            <div>
+              <h3>Blackjack</h3>
+              <a
+                href="https://yuj8fuj6.github.io/basics-blackjack/"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><img
+                  src="blackjack.webp"
+                  alt="Blackjack"
+                  width="200"
+                  height="200"
+                  class="graphics"
+              /></a>
+              <p class="anno-width">Blackjack game with customized theme.</p>
+            </div>
+          </div>
+        </div>
+        <div>
+          <h2>Architecture Projects</h2>
+          <div class="container-row">
+            <div>
+              <h3>Portfolio</h3>
+              <a
+                href="https://issuu.com/yuj8fuj6/docs/2015_"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><img
+                  src="arch-image.jpg"
+                  alt="Arch Projects"
+                  width="200"
+                  height="200"
+                  class="graphics"
+              /></a>
+              <p class="anno-width">
+                Student projects in architectural school.
+              </p>
+            </div>
+            <div>
+              <h3>Art Works</h3>
+              <a
+                href="https://2013ensyu-bc.blogspot.com/search?q=%E8%97%A4%E6%B3%A2%E3%80%80%E5%8B%87%E6%AC%A1%E3%80%80%E3%83%9E%E3%83%AB%E3%82%B3%E3%83%A0"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><img
+                  src="art-project.jpeg"
+                  alt="Art Projects"
+                  width="200"
+                  height="200"
+                  class="graphics"
+              /></a>
+              <p class="anno-width">Art projects in architectural school.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="base-container" id="contact-me">
+        <div class="container">
+          <div>
+            <h1 class="section-header">Contact Me</h1>
+            <p class="sub-header">
+              Looking to know more about me? Feel free to contact me
+            </p>
+            <a class="email-add" href="mailto:yuj.m.fuj91@gmail.com"
+              >via email.</a
+            >
+          </div>
+        </div>
+        <div id="img">
+          <img
+            src="contact-me.jpeg"
+            alt="Contact Me"
+            width="400"
+            height="300"
+          />
+        </div>
+      </div>
+    </section>
   </body>
 </html>
```

Comment:
Overall quite nice html and css here :)! We might want to rethink our naming a bit as names like "base-container" and "container" are a bit hard to tell apart